install/index.php

```diff
@@ -1,276 +1,419 @@
- *
- *  This program is free software: you can redistribute it and/or modify
- *  it under the terms of the GNU General Public License as published by
- *  the Free Software Foundation, either version 3 of the License, or
- *  (at your option) any later version.
- *
- *  This program is distributed in the hope that it will be useful,
- *  but WITHOUT ANY WARRANTY; without even the implied warranty of
- *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- *  GNU General Public License for more details.
- *
- *  You should have received a copy of the GNU General Public License
- *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
-/**
- *  Poweradmin installer
- *
- * @package     Poweradmin
- * @copyright   2007-2010 Rejo Zenger <rejo@zenger.nl>
- * @copyright   2010-2016 Poweradmin Development Team
- * @license     http://opensource.org/licenses/GPL-3.0 GPL
- */
-
-// Dependencies
-require_once dirname(__DIR__) . '/vendor/autoload.php';
-require_once dirname(__DIR__) . '/inc/error.inc.php';
-require_once dirname(__DIR__) . '/inc/i18n.inc.php';
-
-// Constants
-define('LOCAL_CONFIG_FILE', dirname(__DIR__) . '/inc/config.inc.php');
-define('SESSION_KEY_LENGTH', 46);
-
-// Localize interface
-$language = 'en_EN';
-if (isset($_POST['language']) && $_POST['language'] != 'en_EN') {
-    $language = $_POST['language'];
-
-    $locale = setlocale(LC_ALL, $language, $language . '.UTF-8');
-    if ($locale == false) {
-        error(ERR_LOCALE_FAILURE);
-    }
-
-    $gettext_domain = 'messages';
-    if (!function_exists('bindtextdomain')) {
-        die(error('You have to install PHP gettext extension!'));
-    }
-    bindtextdomain($gettext_domain, "./../locale");
-    textdomain($gettext_domain);
-    @putenv('LANG=' . $language);
-    @putenv('LANGUAGE=' . $language);
+
+// Check for debug-mode
+if (array_key_exists('debug', $_GET) && $_GET['debug'] === 'true') {
+    // Enable error-reporting
+    error_reporting(E_ALL);
+    ini_set('display_errors', 1);
+
+    // Set debug-mode constant
+    define('DEBUG', true);
+}
+
+// Starting session
+session_start();
+
+// Set directory-constants
+define('INSTALLER_DIRECTORY', __DIR__ . '/');
+
+// Load autoloader & functions-file
+require_once INSTALLER_DIRECTORY . '../vendor/autoload.php';
+require_once INSTALLER_DIRECTORY . 'includes/functions.php';
+require_once INSTALLER_DIRECTORY . 'includes/validation_functions.php';
+
+// Import used classes
+use Valitron\Validator as V;
+
+// Set array with default-configuration
+$config = [
+    'configFile' => realpath(INSTALLER_DIRECTORY . '../inc') . '/config.inc.php',
+
+    'availableDatabaseDrivers' => Doctrine\DBAL\DriverManager::getAvailableDrivers(),
+    'supportedDatabaseDrivers' => ['pdo_mysql' => 'MySQL', 'pdo_pgsql' => 'PostgreSQL', 'pdo_sqlite' => 'SQLite'],
+
+    'defaultLocale' => 'en_US',
+    'locales' => ['de_DE' => _('German'), 'en_US' => _('English')],
```

Comment:
Hmm, I think only partial the words. The phrases are completely rewritten. As long-term goal we should think about changing the translation-management to store the available locales in databases so the user can install the one he needs.

I've created a organization and project on Transifex. (https://www.transifex.com/poweradmin/)
If you tell me your Username I will add you as Administrator. I think we should wait until someone offers to maintain a language and then add it on Transifex.